### PR TITLE
Add manual testing instructions to DG

### PR DESCRIPTION
**Pull request by dylkaw**
What have you done in this PR?
* Add instructions for manual testing for new features not provided by AB3

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -871,29 +872,229 @@ testers are expected to do more *exploratory* testing.
...
+   3. For other positive test cases, try `editMember` with a valid index and at least one field to edit with a valid input.<br>
+     Expected: Similar to previous.
+
+1. Unsuccessfully editing a member
+    1. Prerequisites: 
+       1. List all members using the `list` command. Multiple members in the list.
+       2. `createMember n/Andy Tan g/Male p/91234567 e/andytan@gmail.com a/10 Kent Ridge Crescent, Singapore 119278 t/logs` has been entered once before.
+    2. Test case: `editMember 1 n/Andy Tan`<br>
+       Expected: No member is edited. Error details shown in the result display. Command box remains the same.
+    3. Other incorrect editMember commands to try: `editMember 1`, `editMember` with valid index, but invalid values in fields to be edited (For invalid values of fields, refer to User Guide).<br>
```
Comment (ph-nathan) [inline docs/DeveloperGuide.md]:
```suggestion
    3. Other incorrect `editMember` commands to try: `editMember 1`, `editMember` with valid index, but invalid values in fields to be edited (For invalid values of fields, refer to User Guide).<br>
```

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -871,29 +872,229 @@ testers are expected to do more *exploratory* testing.
...
+    2. For other negative test cases, try to input invalid values into each field. For unacceptable values of each field, refer to the User Guide.<br>
+       Expected: Similar to previous.
+
+### Editing an event
+
+1. Successfully editing an event
+    1. Prerequisites: List all events using the `list` command. Multiple events in the list.
+    2. Test case: `editEvent 1 l/Pioneer House d/2023-10-25`<br>
+       Expected: Location and date of first event is edited. Details of event edited shown in the result display. Command box is cleared.
+    3. For other positive test cases, try `editMember` with a valid index and at least one field to edit with a valid input.<br>
```
Comment (wr1159) [inline docs/DeveloperGuide.md]:
```suggestion
    3. For other positive test cases, try `editEvent` with a valid index and at least one field to edit with a valid input.<br>
```

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -871,29 +872,229 @@ testers are expected to do more *exploratory* testing.
...
+       Expected: Similar to previous.
+
+1. Unsuccessfully editing an event
+    1. Prerequisites:
+        1. List all events using the `list` command. Multiple events in the list.
+        2. `createEvent n/Run l/NUS Track d/2023-09-20 t/Training` has been entered once before.
+    2. Test case: `editEvent 1 n/Run`<br>
+       Expected: No event is edited. Error details shown in the result display. Command box remains the same.
+    3. Other incorrect editEvent commands to try: `editMember 1`, `editMember` with valid index, but invalid values in fields to be edited (For invalid values of fields, refer to User Guide).<br>
```
Comment (wr1159) [inline docs/DeveloperGuide.md]:
```suggestion
    3. Other incorrect editEvent commands to try: `editEvent 1`, `editEvent` with valid index, but invalid values in fields to be edited (For invalid values of fields, refer to User Guide).<br>
```

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (ph-nathan) [inline docs/DeveloperGuide.md]:
```suggestion
    3. For other positive test cases, try `editEvent` with a valid index and at least one field to edit with a valid input.<br>
```

File: docs/DeveloperGuide.md (hunk 3 above)
Comment (ph-nathan) [inline docs/DeveloperGuide.md]:
```suggestion
    3. Other incorrect editEvent commands to try: `editEvent 1`, `editEvent` with valid index, but invalid values in fields to be edited (For invalid values of fields, refer to User Guide).<br>
```

File: docs/DeveloperGuide.md [hunk 4]
```diff
@@ -871,29 +872,229 @@ testers are expected to do more *exploratory* testing.
...
+### Viewing a member
+
+1. Successfully viewing a member
+   1. Prerequisites: List all members using the `list` command. Multiple members in the list.
+    1. Test case: `viewMember 1`<br>
+       Expected: Event list updated with events of first event. Details of viewed member is shown in the result display. Command box is cleared.
+1. Unsuccessfully viewing a member
+
+    1. Prerequisites: List all members using the `list` command. Multiple members in the list.
+    1. Test case: `deleteEvent 0`<br>
```
Comment (wr1159) [inline docs/DeveloperGuide.md]:
```suggestion
    1. Test case: `viewMember 0`<br>
```

File: docs/DeveloperGuide.md [hunk 5]
```diff
@@ -871,29 +872,229 @@ testers are expected to do more *exploratory* testing.
...
+### Viewing an event
+
+1. Successfully viewing an event
+    1. Prerequisites: List all events using the `list` command. Multiple events in the list.
+    1. Test case: `viewEvent 1`<br>
+       Expected: Member list updated with members of first event. Details of viewed event is shown in the result display. Command box is cleared.
+1. Unsuccessfully viewing an event
+
+    1. Prerequisites: List all events using the `list` command. Multiple events in the list.
+    1. Test case: `deleteEvent 0`<br>
```
Comment (wr1159) [inline docs/DeveloperGuide.md]:
```suggestion
    1. Test case: `viewEvent 0`<br>
```

File: docs/DeveloperGuide.md [hunk 6]
```diff
@@ -871,29 +872,229 @@ testers are expected to do more *exploratory* testing.
...
+    1. Test case: `enrol m/1 e/1 h/3 r/Role: Social Media Manager`<br>
+       Expected: No new enrolment created. Error details shown in the result display. Command box remains the same.
+    2. For other negative test cases, try to input invalid values into each field. For unacceptable values of each field, refer to the User Guide.<br>
+       Expected: Similar to previous.
+
+### Editing an enrolment
+
+1. Successfully editing an enrolment
+    1. Prerequisites: List all members and events using the `list` command. Multiple members and events in the list.
+   1. Prerequisite: `enrol m/1 e/1 h/3 r/Role: Social Media Manager` has been entered once before.
```
Comment (ph-nathan) [inline docs/DeveloperGuide.md]:
```suggestion
    1. Prerequisites: List all members and events using the `list` command. Multiple members and events in the list.
    1. Prerequisite: `enrol m/1 e/1 h/3 r/Role: Social Media Manager` has been entered once before.
```

File: docs/DeveloperGuide.md [hunk 7]
```diff
@@ -871,29 +872,229 @@ testers are expected to do more *exploratory* testing.
...
+    1. Test case: `findMember`<br>
+       Expected: Member list is not updated. Error details shown in the result display. Command box remains the same.
+
+### Find an event
+
+1. Successfully finding an event
+    1. Test case: `findEvent Training`<br>
+       Expected: Event list updated with all events with name matching the word "Training". Number of events with name "Training" shown in the result display. Command box is cleared.
+2. Unsuccessfully finding a member
+    1. Test case: `findMember`<br>
```
Comment (wr1159) [inline docs/DeveloperGuide.md]:
```suggestion
    1. Test case: `findEvent`<br>
```

**Review by wr1159 — CHANGES_REQUESTED** [review]
Added a few comments but other than that LGTM

File: docs/DeveloperGuide.md [hunk 8]
```diff
@@ -871,29 +872,229 @@ testers are expected to do more *exploratory* testing.
...
+       Expected: Member list is not updated. Error details shown in the result display. Command box remains the same.
+
+### Find an event
+
+1. Successfully finding an event
+    1. Test case: `findEvent Training`<br>
+       Expected: Event list updated with all events with name matching the word "Training". Number of events with name "Training" shown in the result display. Command box is cleared.
+2. Unsuccessfully finding a member
```
Comment (ph-nathan) [inline docs/DeveloperGuide.md]:
```suggestion
2. Unsuccessfully finding an event
```

File: docs/DeveloperGuide.md (hunk 7 above)
Comment (ph-nathan) [inline docs/DeveloperGuide.md]:
```suggestion
    1. Test case: `findEvent`<br>
```

File: docs/DeveloperGuide.md [hunk 9]
```diff
@@ -871,29 +872,229 @@ testers are expected to do more *exploratory* testing.
...
+   2. Test case: `undo`<br>
+      Expected: No command is undone. Error details shown in the result display. Command box remains the same.
+
+### Redo
+
+1. Successfully redo a command
+    1. Prerequisite: The last command to be successfully executed is `undo`.
+    2. Test case: `redo`<br>
+       Expected: Previous state before the `undo` will be restored. Command that was redone shown in the result display. Command box is cleared.
+3. Unsuccesfully undo a command
```
Comment (ph-nathan) [inline docs/DeveloperGuide.md]:
```suggestion
3. Unsuccesfully redo a command
```

**Review by ph-nathan — CHANGES_REQUESTED** [review]
Thanks

**Review by wr1159 — APPROVED** [review]
LGTM

**Review by ph-nathan — APPROVED** [review]
Are you a magician? Because whenever I look at you, everyone else disappears.

**Pull request merged**